az-digital/az_quickstart
# WIP: Add ability to use text instead of a logo

## Description
These changes use an implementation of the Drupal 9 system branding
block to allow a text logo or a logo image to be used as a header logo
via the system branding block.

## Related Issue
Closes #771 

## How Has This Been Tested?
This has been tested locally on Lando; however, there are at least three outstanding issues:
1. Remove site_name and site_slogan from the block configuration for the system branding block.
2. Export the block layout settings, so the new system branding block is enabled by default and appears in the correct region.
3. The user documentation should be updated to inform site/content admins of this feature and how to take advantage of it.

## Types of changes


## Checklist:
